code/modules/paperwork/ticketmachine.dm

```diff
@@ -0,0 +1,161 @@
+//Bureaucracy machine!
+//Simply set this up in the hopline and you can serve people based on ticket numbers
+
+/obj/machinery/ticket_machine
+	name = "Ticket machine"
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "ticketmachine"
+	desc = "Please, take a ticket!."
+	density = FALSE
+	maptext_height = 26
+	maptext_width = 32
+	maptext_x = 7
+	maptext_y = 10
+	layer = HIGH_OBJ_LAYER
+	var/ticket_number = 0 //Increment the ticket number whenever the HOP presses his button
+	var/current_number = 0 //What customer are we currently serving?
+	var/max_number = 999 //To stop the text going fucky. At this point, you need to refill it.
+	var/cooldown = 50 //Small cooldown, stops the clown from immediately breaking it.
+	var/ready = TRUE
+	var/id = "ticket_machine_default" //For buttons
+
+/obj/machinery/ticket_machine/emag_act(mob/user) //Emag the ticket machine to dispense burning tickets, as well as randomize its customer number to destroy the HOP's mind.
+	if(obj_flags & EMAGGED)
+		return
+	to_chat(user, "<span class='warning'>You overload [src]'s bureaucratic logic circuitry to its MAXIMUM setting.</span>")
+	ticket_number = rand(0,999)
+	obj_flags |= EMAGGED
+
+/obj/machinery/ticket_machine/Initialize()
+	. = ..()
+	update_icon()
+
+/obj/machinery/ticket_machine/proc/increment()
+	if(current_number >= ticket_number)
+		return
+	playsound(src, 'sound/misc/announce_dig.ogg', 50, 0)
```

Comment:
announce_dig?

Reply:
Alternative announcement sound that's there for some reason, seemed un-used elsewhere